src/docs/Chakra.mdx

```diff
@@ -81,29 +81,30 @@ const Application = (props) => {
   language="tsx"
 />
 
-### Chakra Components in Storybook
+### Disabling Global Styles
```

Comment:
I feel this should be a separate page under the `Development Guide`.

Reply:
Good idea.